### PR TITLE
Homework3 git

### PR DESCRIPTION
**Pull request by Iliailineya**
Create tests

### CODE REVIEW

File: src/test/java/service/AccountServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AccountServiceTest {
+
+    @Test
+    void testFilterAccountsByBalance() {
+        List<Account> accounts = createTestAccounts();
+        AccountService accountService = new AccountService();
```
Comment (YeaaaaH) [inline src/test/java/service/AccountServiceTest.java]: every test has accountService  initialization, put it in @BeforeAll setup()/init()

File: src/test/java/service/AccountServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (YeaaaaH) [inline src/test/java/service/AccountServiceTest.java]: don't use wildcard imports

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need to fix

**Comment by Iliailineya** [comment]
Done

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**